.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Bug Report 🐛
+about: Something isn't working as expected? Report it here.
+---
+
+<!--
```

Comment:
I suspect people will be confused, so can you add "(You can delete this text here)" or something along those lines?

Reply:
Instead of spamming you externally, I'll reply here in case anyone wants to chime in!

Proposals I can think of are:
```
# Description
(DELETE THIS TEXT PLEASE FOR ALL THAT IS HOLY) Please describe your bug. :)
```
Pros/Cons:
\+ Hopefully straightforward
\- Will look awful if they don't delete the line

```
# Description
<!-- Please describe your bug. :) -->
```
Pros/Cons:
\+ Doesn't look awful if they don't delete it
\- If they put their text within the comments they will submit an empty issue

```
**Description:**
```
Pros/Cons:
\+ Least room for error
\- Not a lot of guidance

---
In the wild:

[CockroachDB](https://raw.githubusercontent.com/cockroachdb/cockroach/master/.github/ISSUE_TEMPLATE/bug_report.md) uses the first option to a less extreme.

[numpy](https://raw.githubusercontent.com/numpy/numpy/master/.github/ISSUE_TEMPLATE.md) and [Electron](https://raw.githubusercontent.com/electron/electron/master/.github/ISSUE_TEMPLATE/Bug_report.md) use the second.

[ESLint](https://raw.githubusercontent.com/eslint/eslint/master/.github/ISSUE_TEMPLATE/BUG_REPORT.md), [React](https://raw.githubusercontent.com/facebook/react/master/.github/ISSUE_TEMPLATE.md), and [Kubernetes](https://raw.githubusercontent.com/kubernetes/kubernetes/master/.github/ISSUE_TEMPLATE/enhancement.md) have taken the third approach.

I don't really want to rank any of these in terms of their "goodness" or appeal to populism, but these were some examples I was able to find. I think all of these repositories are directed towards more advanced users than we sometimes see, but I don't think there's _too_ much issue in confusing people; I think a user that is less inclined to submit an issue either won't, because they already don't have a GitHub account, or they will open a new issue and see the Discord and post there instead, or -- in a final case that should be ignored -- they will absolutely bypass anything we throw at them regardless of how strong the guardrails are.

Reply:
Hmm.  Yeah, I think the worry that maybe somebody will submit their entire bug inside the comment block is a bit real.  Maybe your original is fine, here.